install_sct

```diff
@@ -89,15 +88,13 @@ install_refresh ()
     echo
     echo "" >> ${THE_BASHRC}
     echo "# SPINALCORDTOOLBOX (installed on $(date +%Y-%m-%d\ %H:%M:%S))" >> ${THE_BASHRC}
-    echo "export PATH=${SCT_DIR}/${BIN_DIR}:\$PATH" >> ${THE_BASHRC}
-    echo "source ${SCT_DIR}/${BIN_DIR}/sct_env" >> ${THE_BASHRC}
+    echo "PATH+=\":${SCT_DIR}/${BIN_DIR}\"" >> ${THE_BASHRC}
```

Comment:
any reason for appending sct at the end instead of at the beginning of PATH? if users re-install multiple versions of SCT overtime, the latter approach ensures to point to the correct SCT install (we had related issues in the past)

Reply:
Forgot about that. I'll revert this portion.

Reply:
Fixed in subsequent commit.